CHANGELOG..md

```diff
@@ -0,0 +1,108 @@
+the Github issue reference in the following format:
+
+* (<tag>) \#<issue-number> message
+
+The issue numbers will later be link-ified during the release process so you do
+not have to worry about including a link manually, but you can if you wish.
+
+Types of changes (Stanzas):
+
+"Features" for new features.
+"Improvements" for changes in existing functionality.
+"Deprecated" for soon-to-be removed features.
+"Bug Fixes" for any bug fixes.
+"Client Breaking" for breaking CLI commands and REST routes used by end-users.
+"API Breaking" for breaking exported APIs used by developers building on SDK.
+"State Machine Breaking" for any changes that result in a different AppState
+given same genesisState and txList.
+Ref: https://keepachangelog.com/en/1.1.0/
+-->
+# Changelog
+
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) after the `v1.0.1-rc-testnet` release.
+
+## Unreleased
+
+## v1.0.1-rc-testnet Update Wasm
+* [568](https://github.com/quasar-finance/quasar/pull/568) Adding the was capability support on chain for x/wasm osmosis,cosmwasm_1_1,cosmwasm_1_2,cosmwasm_1_4. and bump the wasm version
+
+
+## v1.0.7-milktia-tia-patch
+* [560](https://github.com/quasar-finance/quasar/pull/560) Change the cl vault withdraw events to exist under their own event
+
+## v1.0.6-milktia-tia-patch
+
+* [557](https://github.com/quasar-finance/quasar/pull/557) Paginate Cl vault distribute rewards
+
+## v1.0.7-cl
+* [549](https://github.com/quasar-finance/quasar/pull/549) Validate and rebuildCL Tick caches of the CL vault
+* [555](https://github.com/quasar-finance/quasar/pull/555) Adds pool migration to migrate from to pools that only differ fee tier
+
+## 1.0.6-cl
+* [508](https://github.com/quasar-finance/quasar/pull/508) Build the CL tick cache in CL vault instantiation
+* [512](https://github.com/quasar-finance/quasar/pull/512) Bump Osmosis-std to 0.19.2
+* [524](https://github.com/quasar-finance/quasar/pull/524) Bump interchain e2e tests and CL vault to Osmosis v20 
+* [493](https://github.com/quasar-finance/quasar/pull/493) Add an airdrop contract
+* [537](https://github.com/quasar-finance/quasar/pull/537) Change the CL vault initial mint to lower amounts
+
+
+## 1.0.5-cl
+* [511](https://github.com/quasar-finance/quasar/pull/511) Add more balance queries to the CL vault
+* [513](https://github.com/quasar-finance/quasar/pull/513) Add swaps using less than 100% of the swap amount and twap to the CL vault
+
+## 1.0.4-cl
+* [503](https://github.com/quasar-finance/quasar/pull/503) Set max slippage per rerange operation instead of in the vault config
+* [501](https://github.com/quasar-finance/quasar/pull/501) Use any free balance of the CL vault in reranging
+* [505](https://github.com/quasar-finance/quasar/pull/505) Add empty migrate entrypoint
+
```

Comment:
Tag 1.0.3-cl is missing.

Reply:
Added tag 1.0.3-cl under yanked, since we decided not to actually release it onchain